docs/root/intro/arch_overview/ssl.rst

```diff
@@ -30,6 +30,23 @@ Underlying implementation
 Currently Envoy is written to use `BoringSSL <https://boringssl.googlesource.com/boringssl>`_ as the
 TLS provider.
 
+.. _arch_overview_ssl_fips:
+
+FIPS
+----
+
+BoringSSL can be built in a
+`FIPS-compliant mode <https://boringssl.googlesource.com/boringssl/+/master/crypto/fipsmodule/FIPS.md>`_,
+following the build instructions from the `Security Policy for BoringCrypto module
+<https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp3318.pdf>`_,
+using ``--define boringssl=fips`` Bazel option. This option is only available on Linux-x86_64.
```

Comment:
Consider changing to something like `--define boringssl_fips=true` so that if there are other boringssl-related compile-time flags at some other time, they're easier to add.

Reply:
The reason for `--define boringssl=fips` was to re-use the same flag for builds without BoringSSL in the future, i.e. `--define boringssl=disabled` (basically, having  `enabled` (default), `fips` and `disabled` as a valid options).